### ISSUE TITLE
logger_level at NONE cliupdate.php issues DEBUG output

### ISSUE THREAD

**Issue by PresGas** [SSilence/selfoss#1077]
I am hosting a selfoss 2.18 installation on nearlyfreespeech.net. When I set the logger_level to NONE, cron outputs for the cliupdate.php seem to default to DEBUG. All the other options for logger_level run at the appropriate levels.

My cron line is like so:

`/usr/local/bin/php7.2 /home/public/cliupdate.php`

Let me know if you need other info.

**Comment by jtojnar** [comment]
Hmm, apparently Monolog uses a fallback handler when we do not add any handlers: https://github.com/Seldaek/monolog/blob/4d5b7e6ba1127789c7ff59d6f762298eaa29787f/src/Monolog/Logger.php#L294-L296

This is fixed in https://github.com/SSilence/selfoss/commit/2395a12a357efd4a2f22edd37385f060a1aa52b4, thank you for reporting this.

**Comment by PresGas** [comment]
Awesome and thank you for the fast turnaround!

Any "rough" target dates for 2.19 release?

**Comment by jtojnar** [comment]
We are blocked on https://github.com/SSilence/selfoss/pull/1014 and I am not sure when I will have large enough time window to debug the two remaining issues so I am unable to give any time frame.

However, the development builds are usually more reliable than stable releases since they include many bug fixes; I would recommend you to just use https://bintray.com/fossar/selfoss/download_file?file_path=selfoss-2.19-a5841a5.zip

**Comment by GutHib** [comment]
Oddly I'm seeing the same behavior with the latest stable version 2.19. Updated today, and the servers HD has been filling like crazy (caught it at 99% and a log file of 8 GB, even though logger_level is set to NONE).

Sample from the logs...

`[2023-03-28 23:15:02] selfoss.DEBUG: set cookie on www.XXXXXXXX.com/news/ expiring in 2592000 seconds  
[2023-03-28 23:15:02] selfoss.DEBUG: session does not contain valid auth  
[2023-03-28 23:15:03] selfoss.DEBUG: Creating database connection {"dsn":"mysql:host=localhost; port=3306; dbname=XXXXXXXX_db"} 
[2023-03-28 23:15:03] selfoss.DEBUG: Establishing MySQL database connection  
[2023-03-28 23:15:03] selfoss.DEBUG: ---  `

Seems like more or less the entire feed content is ending up in the logs, too.

**Comment by jtojnar** [comment]
@GutHib just to double check, what do you see if you modify selfoss as follows:

```patch
--- a/src/common.php
+++ b/src/common.php
@@ -48,6 +48,7 @@ $f3->set('BASEDIR', BASEDIR);
 
 $configuration = new Configuration(__DIR__ . '/../config.ini', $_ENV);
 
+var_dump($configuration->loggerLevel, $configuration->debug);
 $f3->set('DEBUG', $configuration->debug);
 $f3->set('cache', $configuration->cache);
 
```

Do you have this line in your `src/common.php`?

https://github.com/fossar/selfoss/blob/707cbd14b2dd09a6ab7df4426b265857802c6328/src/common.php#L247-L248

**Comment by jtojnar** [comment]
> With logging set to "NONE" and the extra var_dump line it will continue. Or do you want me to check if the content of the log file is any different?

`var_dump` line should print the values in the browser to make sure it is not being overridden somehow.

**Comment by GutHib** [comment]
The output is:

`string(0) "" int(0)`

...when set to "NONE". When set to "ERROR", it is:

`string(5) "ERROR" int(0)`

**Comment by GutHib** [comment]
Works now after putting 'NONE' in single quotes.

I noticed the same with the password, had to put it in (single) quotes after the update.

**Comment by jtojnar** [comment]
@GutHib Oh, I forgot this was an issue. That was fixed with ba9339372a7bc0678c6c1f74336406ab1bbb4ecb.
